01_Data/src/layers/sequelize/repository/Authorization.ts

```diff
@@ -44,14 +40,16 @@ export class SequelizeAuthorizationRepository extends SequelizeRepository<Author
     }
 
     const savedAuthorizationModel = await Authorization.findOne({
-      include: [{
```

Comment:
you merged a lint-fix branch into 1.3.2. Why is this formatting change still happening? We need to be more strict with this, we can't have every pr contain irrelevant formatting changes.

Reply:
💯 yeah let me try running lint again, I am not sure what happened here...

Reply:
I created this PR here - https://github.com/citrineos/citrineos-core/pull/208 to lint-fix... I am not sure how changes got in without the lint catching these, but I do see this change in the lint PR so I think once that one is merged, then this PR should no longer show this 👍